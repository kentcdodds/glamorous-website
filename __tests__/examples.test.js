/* global it, expect, describe */

import React from 'react'
import renderer from 'react-test-renderer'
import {matcher, serializer} from 'jest-glamor-react'
import Examples from '../pages/examples'

expect.addSnapshotSerializer(serializer)
expect.extend(matcher)

describe('Testing Examples', () => {
  it('Renders!', () => {
    const component = renderer.create(<Examples/>)
    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
})
