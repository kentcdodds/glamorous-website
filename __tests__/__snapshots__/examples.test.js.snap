// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Examples Renders! 1`] = `
<div>
  <div
    className="css-14e33nr"
  >
    <div
      className="css-151tlpa"
    >
      <nav
        className="css-63oe3q"
      >
        <ul
          className="css-35577l"
        >
          <li
            className="css-wfyxwh"
          >
            <svg
              enableBackground="new 0 0 64 64"
              onClick={[Function]}
              viewBox="0 0 64 64"
              width={20}
              xmlns="http://www.w3.org/2000/svg"
            >
              <g
                fill="#ED4C5C"
              >
                <path
                  d="M20.8 6.4 40.9 26.5 29.7 37.7 2.2 10.2z"
                />
                <path
                  d="m10.6 5.8c-5.2.9-9 2.7-8.6 4.1.5 1.4 5.1 1.8 10.4.9 5.2-1 9.1-2.8 8.6-4.2-.5-1.4-5.2-1.7-10.4-.8"
                />
              </g>
              <path
                d="m10.8 5.8c-4.9.8-8.5 2.6-8 3.9.5 1.3 4.8 1.6 9.7.8 4.9-.8 8.5-2.6 8-3.8-.5-1.4-4.8-1.8-9.7-.9"
                fill="#ff8080"
              />
              <path
                d="m22.6 16.5h18.2v23.8h-18.2z"
                fill="#e8e8e8"
                transform="matrix(.7071-.7071.7071.7071-10.7928 30.6765)"
              />
              <path
                d="m43.7 47h19v5.9h-19z"
                fill="#555b61"
                transform="matrix(.707-.7072.7072.707-19.7271 52.2523)"
              />
              <path
                d="m27.2 30.7h19v5.5h-19z"
                fill="#b2c1c0"
                transform="matrix(.7069-.7073.7073.7069-12.8952 35.7644)"
              />
              <path
                d="m34.8 29.8h21v24.4h-21z"
                fill="#3e4347"
                transform="matrix(.7071-.7071.7071.7071-16.45 44.3334)"
              />
            </svg>
          </li>
          <li
            className="css-wfyxwh"
          >
            <a
              className="css-zr73hg"
              href="https://github.com/paypal/glamorous"
            >
              GitHub
            </a>
          </li>
          <li
            className="css-wfyxwh"
          >
            <a
              className="css-zr73hg"
              onClick={[Function]}
            >
              Guides
            </a>
          </li>
          <li
            className="css-wfyxwh"
          >
            <a
              className="css-zr73hg"
              onClick={[Function]}
            >
              Examples
            </a>
          </li>
          <li
            className="css-wfyxwh"
          >
            <a
              className="css-zr73hg"
              onClick={[Function]}
            >
              Docs
            </a>
          </li>
        </ul>
      </nav>
    </div>
    <div
      className="css-2y5bqi"
    >
      <div
        className="css-1nvbmsg"
      >
        <div
          className="css-17186zx"
        >
          <div
            className="css-cuyszr"
          >
            {
            <div
              className="css-ehn2ke"
            >
              <svg
                enableBackground="new 0 0 64 64"
                viewBox="0 0 64 64"
                xmlns="http://www.w3.org/2000/svg"
              >
                <g
                  fill="#ED4C5C"
                >
                  <path
                    d="M20.8 6.4 40.9 26.5 29.7 37.7 2.2 10.2z"
                  />
                  <path
                    d="m10.6 5.8c-5.2.9-9 2.7-8.6 4.1.5 1.4 5.1 1.8 10.4.9 5.2-1 9.1-2.8 8.6-4.2-.5-1.4-5.2-1.7-10.4-.8"
                  />
                </g>
                <path
                  d="m10.8 5.8c-4.9.8-8.5 2.6-8 3.9.5 1.3 4.8 1.6 9.7.8 4.9-.8 8.5-2.6 8-3.8-.5-1.4-4.8-1.8-9.7-.9"
                  fill="#ff8080"
                />
                <path
                  d="m22.6 16.5h18.2v23.8h-18.2z"
                  fill="#e8e8e8"
                  transform="matrix(.7071-.7071.7071.7071-10.7928 30.6765)"
                />
                <path
                  d="m43.7 47h19v5.9h-19z"
                  fill="#555b61"
                  transform="matrix(.707-.7072.7072.707-19.7271 52.2523)"
                />
                <path
                  d="m27.2 30.7h19v5.5h-19z"
                  fill="#b2c1c0"
                  transform="matrix(.7069-.7073.7073.7069-12.8952 35.7644)"
                />
                <path
                  d="m34.8 29.8h21v24.4h-21z"
                  fill="#3e4347"
                  transform="matrix(.7071-.7071.7071.7071-16.45 44.3334)"
                />
              </svg>
            </div>
            }
          </div>
          <div
            className="css-6l3qzw"
          >
            glamorous
          </div>
        </div>
      </div>
      <p
        className="css-nil"
      >
        edit in pages/examples.js
      </p>
    </div>
    <footer
      className="css-kv69lr"
    >
      <div
        className="css-ubp1jq"
      >
        Made with ðŸ’™ by
         
        <a
          className="css-zr73hg"
          href="https://github.com/paypal"
        >
          PayPal
        </a>
        .
      </div>
    </footer>
  </div>
</div>
`;
